Products/Zuul/catalog/model_catalog_tool.py

```diff
@@ -282,9 +282,9 @@ def search(self, types=(), start=0, limit=None, orderby='name',
                        fields we need to retrieve the faster the query will be
         @param facets_for_field: Field for which we want to retrieve its facets
         """
-        available_indexes = self.model_catalog_client.get_indexes()
+        indexed, not_indexed = self.model_catalog_client.get_indexes()
```

Comment:
Same as above.

Reply:
I also prefer not to change variable names if possible to minimize changes as much as possible